mw-server/internal/db/query/job_dones.query.sql

```diff
@@ -87,9 +87,9 @@ SELECT
         SELECT 1
         FROM mentor_users_ways
         WHERE mentor_users_ways.way_uuid = ways.uuid
-        AND mentor_users_ways.user_uuid = @user_uuid
+          AND mentor_users_ways.user_uuid = @user_uuid
     ) OR ways.owner_uuid = @user_uuid AS is_permission_given
 FROM ways
-INNER JOIN day_reports ON ways.uuid = day_reports.way_uuid
-INNER JOIN job_dones ON job_dones.day_report_uuid = day_reports.uuid
-WHERE job_dones.uuid = @job_dones_uuid;
+         INNER JOIN day_reports ON ways.uuid = day_reports.way_uuid
```

Comment:
choose consistent tabulation: I recommend 4 as everywhere in the project. Check everywhere